charmhelpers/contrib/openstack/audits/__init__.py

```diff
@@ -39,7 +39,7 @@ def audit(*args):
     deployed system that matches the given configuration
 
     :param args: List of functions to filter tests against
-    :type args: List[Callable(Config)]
+    :type args: List[Callable(dict)]
```

Comment:
I think it needs to be square brackets:  https://docs.python.org/3/library/typing.html#callable